doc/user/COMMANDS.md

```diff
@@ -79,6 +79,11 @@ eg: `set_hdri /path/to/file.hdr`.
 `add_files_or_set_hdri [path/to/file1] [path/to/file2]`: A specific command that will process each files and either, `set_hdri` if the provided file uses a recognised HDR extension or `add_files` otherwise. Take one or more files as arguments.
 eg: `add_files_or_set_hdri /path/to/dragon.vtu /path/to/file.hdr`.
 
+## Command Script (`--command-script`)
+
+F3D provides a feature to load and execute commands from a script file using the `--command-script` CLI option. This allows users to automate a sequence of commands by listing them in a plain text file.
+eg: `--command-script path/to/command_script.txt`.
+
```

Comment:
Add a simple script example, the point is to show the commands are separated by new lines.